src/bundle/Resources/public/scss/_main-menu.scss

```diff
@@ -0,0 +1,233 @@
+.ibexa-main-menu {
+    display: flex;
+    height: 100%;
+    min-height: calc(100vh - #{calculateRem(72px)});
+    background-color: $ibexa-color-black;
+
+    &__navbar {
+        position: relative;
+        overflow: hidden;
+
+        &--first-level {
+            width: calculateRem(276px);
+            background-color: $ibexa-color-dark;
+            transition: all $ibexa-admin-transition-duration;
+
+            .ibexa-main-menu__items-list {
+                height: calc(100vh - #{calculateRem(72px)});
+            }
+
+            &.ibexa-main-menu__navbar--collapsed {
+                width: calculateRem(72px);
+            }
+        }
+
+        &--second-level {
+            width: calculateRem(220px);
+            background-color: rgba($ibexa-color-dark, 0.8);
```

Comment:
should we be using rgba instead of one of our defined colors?

Reply:
> should we be using rgba instead of one of our defined colors?

yes, because finally background is blend two colors: main menu and second level menu background